packages/release/netdog.template

```diff
@@ -0,0 +1,6 @@
+{{#if settings.dns.name-servers}}
```

Comment:
Nit: For the lack of a better place to leave this comment:

The subject for this commit doesn't include a "subsystem", in other commits `netdog` was used.